optimum/bettertransformer/transformation.py

```diff
@@ -157,7 +157,17 @@ def transform(
             remove_hook_from_module(model, recurse=True)
 
         if keep_original_model:
-            model_fast = deepcopy(model)
+            model = model.requires_grad_(False)
+            try:
+                model_fast = deepcopy(model)
+            except RuntimeError:
+                raise ValueError(
+                    f"The model {model.__class__.__name__} does not support `deepcopy` operation that is "
+                    " internally used to create a copy of the original model when using"
+                    " `keep_original_model=True`. Please run the conversion with "
+                    " `keep_original_model=Fale` and create a new copy of the original"
```

Comment:
```suggestion
                    " `keep_original_model=False` and create a new copy of the original"
```

So you found the issue in the end? For wav2vec2 we can't use `keep_original_model=True`?

Reply:
The issue seems to be related to `deepcopy` I didn't managed to fix this yet :/

Reply:
Let's maybe keep this for now, the error message seems quite explicit

Reply:
all good